src/Symfony/Component/Notifier/Bridge/Expo/ExpoOptions.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Expo;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Imad ZAIRIG <https://github.com/zairigimad>
+ *
+ * @see https://docs.expo.dev/push-notifications/sending-notifications/
+ *
+ * @experimental in 5.4
```

Comment:
We cannot introduce experimental features in an LTS release, please remove this line

Reply:
I removed experimental